server/@types/bapv.d.ts

```diff
@@ -64,16 +64,24 @@ export type PrisonerProfilePage = {
   activeAlerts: PrisonerAlertItem[]
   activeAlertCount: number
   flaggedAlerts: Alert[]
+  prisonerDetails: {
```

Comment:
Incorporating `PrisonerDetails` and `BAPVVisitBalances` types into `PrisonerProfilePage` for simplicity and because they aren't re-used outside the profile page.